### DISCUSSION TITLE
.reに画像を挿入

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1828] [Q&A] [answered]
.reに画像を挿入することは可能ですか？
および.reへの入力方法に関するドキュメント？

![image](https://user-images.githubusercontent.com/49851974/177449902-c69f48bd-693c-4dc8-bcca-4d2196c5c671.png)

**Comment by kmuto** [comment, marked as answer]
Wordなどと異なり、WYSIWYGではありません(VS Codeの拡張でも編集画面上で画像を入れてどうこうというスタイルではないはず)。TeXやMarkdown、HTMLなどと同じくreはテキストファイルであり、画像はimagesフォルダに置いてそれをreからマークアップで参照する、という手法です。
reの記法については、docにあるquickstartやformatなどを参考にしてください。(英語ドキュメントのほうはたぶんいまいちなので、日本語を読むのに支障がなければja付きの日本語ドキュメント側をお勧めします。)
